.buildkite/pipeline.yml

```diff
@@ -1,6 +1,36 @@
 steps:
-  - commands:
+  - label: ":hammer: running tests"
+    commands:
       - "make ci"
-    name: ":hammer: build"
+        #- "make test" #currently no ct defined, causes build error
+    key: "tests"
+    agents:
+      queue: "erlang"
+
+  - wait
+
+  - label: ":debian: build deb"
+    commands:
+      - "git checkout ${BUILDKITE_TAG}"
+      - "make ci"
```

Comment:
Why run CI again here?

Reply:
I guess I'm still thinking from my previous CI system where you'd want to execute tests before packaging always - but in this case it doesn't make sense.